app/main/menus.js

```diff
@@ -29,8 +24,12 @@ function macMenuAppium () {
       label: i18n.t('About Appium'),
       click () {
         dialog.showMessageBox({
```

Comment:
I'd rather extract this dialog into separate module to avoid unnecessary duplication